mvn -Pdocker install fails on OS X due to sed backslash.

I was unable to get the maven target to run correctly on the dev branch, so I checked out `master` and was able to start the db and webserver. However, something is still broken. When I attempt to login as admin I get an invalid username or password message and see this in the tomcat logs:

`secshep_tomcat | DEBUG 172.19.0.1 from ?.?.?.? @ 02 Jan 22:17:35,310 [https-openssl-nio-8443-exec-2] (Getter.java:70) - $$$ Getter.authUser $$$
secshep_tomcat | DEBUG 172.19.0.1 from ?.?.?.? @ 02 Jan 22:17:35,312 [https-openssl-nio-8443-exec-2] (Getter.java:71) - userName = admin
secshep_tomcat | DEBUG 172.19.0.1 from ?.?.?.? @ 02 Jan 22:17:35,349 [https-openssl-nio-8443-exec-2] (Getter.java:80) - Gathering userFind ResultSet
secshep_tomcat | ERROR 172.19.0.1 from ?.?.?.? @ 02 Jan 22:17:35,352 [https-openssl-nio-8443-exec-2] (Getter.java:153) - Login Failure: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: PROCEDURE core.userFind does not exist
secshep_tomcat | DEBUG 172.19.0.1 from ?.?.?.? @ 02 Jan 22:17:35,353 [https-openssl-nio-8443-exec-2] (Getter.java:158) - $$$ End authUser $$$`

I would really appreciate guidance on how to get this running correctly in docker.

Here's what happens when I run `mvn -Pdocker clean install -DskipTests` on the `dev` branch:

`
[INFO] --- exec-maven-plugin:1.6.0:exec (convert-sql-scripts) @ owaspSecurityShepherd ---
sed: 1: "docker/mysql/coreSchema ...": extra characters at the end of d command
sed: 1: "docker/mysql/moduleSche ...": extra characters at the end of d command
sed: 1: "docker/mysql/coreSchema ...": extra characters at the end of d command
sed: 1: "docker/mysql/moduleSche ...": extra characters at the end of d command
sed: 1: "docker/mysql/coreSchema ...": extra characters at the end of d command
sed: 1: "docker/mysql/moduleSche ...": extra characters at the end of d command
[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
    at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:404)
    at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:166)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:804)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:751)
...
`

--- a/.gitignore
+++ b/.gitignore
@@ -3,3 +3,5 @@
 /.project
 /bin/
 /.settings/
+docker/*/*.sql
+docker/*/*.sql.bak

--- a/.gitignore
+++ b/.gitignore
@@ -4,4 +4,3 @@
 /bin/
 /.settings/
 docker/*/*.sql
-docker/*/*.sql.bak

--- a/.gitignore
+++ b/.gitignore
@@ -4,3 +4,4 @@
 /bin/
 /.settings/
 docker/*/*.sql
+docker/*/*.sql-e

--- a/docker/scripts/convert-sql-scripts.sh
+++ b/docker/scripts/convert-sql-scripts.sh
@@ -3,11 +3,11 @@
 core="docker/mysql/coreSchema.sql"
 module="docker/mysql/moduleSchemas.sql"
 
-sed -i '' 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${core}
-sed -i '' 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${module}
+sed -i'' -e 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${core}
+sed -i'' -e 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${module}
 
-sed -i '' 's/-- \$\$/\$\$/g' ${core}
-sed -i '' 's/-- \$\$/\$\$/g' ${module}
+sed -i'' -e 's/-- \$\$/\$\$/g' ${core}
+sed -i'' -e 's/-- \$\$/\$\$/g' ${module}
 
-sed -i '' 's/-- DELIMITER \;/DELIMITER \;/g' ${core}
-sed -i '' 's/-- DELIMITER \;/DELIMITER \;/g' ${module}
+sed -i'' -e 's/-- DELIMITER \;/DELIMITER \;/g' ${core}
+sed -i'' -e 's/-- DELIMITER \;/DELIMITER \;/g' ${module}

--- a/docker/scripts/convert-sql-scripts.sh
+++ b/docker/scripts/convert-sql-scripts.sh
@@ -3,11 +3,11 @@
 core="docker/mysql/coreSchema.sql"
 module="docker/mysql/moduleSchemas.sql"
 
-sed -i .bak 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${core}
-sed -i .bak 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${module}
+sed -i '' 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${core}
+sed -i '' 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${module}
 
-sed -i .bak 's/-- \$\$/\$\$/g' ${core}
-sed -i .bak 's/-- \$\$/\$\$/g' ${module}
+sed -i '' 's/-- \$\$/\$\$/g' ${core}
+sed -i '' 's/-- \$\$/\$\$/g' ${module}
 
-sed -i .bak 's/-- DELIMITER \;/DELIMITER \;/g' ${core}
-sed -i .bak 's/-- DELIMITER \;/DELIMITER \;/g' ${module}
+sed -i '' 's/-- DELIMITER \;/DELIMITER \;/g' ${core}
+sed -i '' 's/-- DELIMITER \;/DELIMITER \;/g' ${module}

--- a/docker/scripts/convert-sql-scripts.sh
+++ b/docker/scripts/convert-sql-scripts.sh
@@ -3,11 +3,11 @@
 core="docker/mysql/coreSchema.sql"
 module="docker/mysql/moduleSchemas.sql"
 
-sed -i 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${core}
-sed -i 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${module}
+sed -i .bak 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${core}
+sed -i .bak 's/-- DELIMITER \$\$/DELIMITER \$\$/g' ${module}
 
-sed -i 's/-- \$\$/\$\$/g' ${core}
-sed -i 's/-- \$\$/\$\$/g' ${module}
+sed -i .bak 's/-- \$\$/\$\$/g' ${core}
+sed -i .bak 's/-- \$\$/\$\$/g' ${module}
 
-sed -i 's/-- DELIMITER \;/DELIMITER \;/g' ${core}
-sed -i 's/-- DELIMITER \;/DELIMITER \;/g' ${module}
+sed -i .bak 's/-- DELIMITER \;/DELIMITER \;/g' ${core}
+sed -i .bak 's/-- DELIMITER \;/DELIMITER \;/g' ${module}

